src/core/lib/event_engine/tcp_socket_utils.cc

```diff
@@ -305,9 +348,17 @@ absl::StatusOr<std::string> ResolvedAddressToURI(
   if (resolved_address.size() == 0) {
     return absl::InvalidArgumentError("Empty address");
   }
-  auto scheme = GetScheme(resolved_address);
+  EventEngine::ResolvedAddress resolved_addr = resolved_address;
```

Comment:
The name looks confusing with `resolved_address`.

Reply:
Changed it name simply to addr